x-pack/plugins/screenshotting/public/plugin.tsx

```diff
@@ -21,26 +21,24 @@ interface SetupDeps {
 
 export class ScreenshottingPlugin implements Plugin<void, void, SetupDeps> {
   setup({ application }: CoreSetup, { screenshotMode }: SetupDeps) {
-    if (!screenshotMode.isScreenshotMode()) {
-      return;
```

Comment:
Early return is a good practice that reduces cyclomatic complexity.

Reply:
I absolutely agree, and use it and promote, too; however, I don't think it is a good choice for plugin lifecycles. I think, for plugin life-cycles, it is better to always have on *ReturnStatement* at the bottom of the life-cycle function, where everyone expects it to be.